docs/source/_topic_map.yml

```diff
@@ -14,8 +14,8 @@ Topics:
         File: installing
       - Name: Quickstart
         File: quickstart
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
+      - Name: Setting up the hypervisor drivers
```

Comment:
why the rename from docker machine drivers to hypervisor drivers. We had a reason to stick to the term "Docker machine drivers". This is the proper term and also the term one needs to search for for finding appropriate information.